handsontable/test/e2e/core/setDataAtCell.spec.js

```diff
@@ -106,4 +106,48 @@ describe('Core.setDataAtCell', () => {
 
     expect(beforeChange).toHaveBeenCalledWith([[0, 1, 'foo', 'foo2']], 'multiple-change');
   });
+
+  it('should keep boolean value in the cell after copy and paste checkbox', () => {
+    handsontable({
+      data: [
+        [true],
+        [false],
+      ],
+      columns: [
+        {
+          data: 0,
+          type: 'checkbox'
+        },
+      ],
+    });
+
+    expect(getDataAtCell(1, 0)).toBe(false);
+
```

Comment:
I guess, missing `setData` call before expectation?
```suggestion

    setDataAtCell(1, 0, false);

    expect(getDataAtCell(1, 0)).toBe(false);

```

Reply:
updated